components/property_info.rst

```diff
@@ -0,0 +1,432 @@
+.. index::
+    single: PropertyInfo
+    single: Components; PropertyInfo
+
+The PropertyInfo Component
+==========================
+
+The PropertyInfo component provides the functionality to get information about class properties using metadata of
+popular sources.
```

Comment:
this paragraph should be indented with 4 spaces to be consistent with other component docs.